### DISCUSSION TITLE
It looks postdoc.dev is down.

### DISCUSSION THREAD

**Discussion by kaliPrasad6400** [PostDocJS/postdoc#72] [General]
postdoc.dev seems down
@beatfactor Can you please look into this.

**Comment by beatfactor** [comment]
the website is not ready yet. it will be online soon.

**Comment by kaliPrasad6400** [comment]
Currently I am working on NightWatchJS docs. Which is dependent on PostDoc.

Thanks for your response.

**Reply by beatfactor** [reply]
Ok, there's an update coming soon for the nightwatch docs. I will send a PR in the next few days.

**Reply by kaliPrasad6400** [reply]
I am currently planning to work on docs as part of GSOC-24.
If possible, Can you explain what the PR is about? 

Sorry to disturb.

**Reply by beatfactor** [reply]
It's mostly some changes to ensure compatibility with the latest changes in Postdoc, mostly around api docs generation. So it's actual docs changes.
